scripts/Gripper.py

```diff
@@ -346,7 +373,9 @@ def reboot(self):
             start_time = time.perf_counter()
             while time.perf_counter() < start_time + 0.5:
                 pass
```

Comment:
just use a sleep command here - why do the servos reboot then init the gripper? why not just init the gripper?